main.go

```diff
@@ -88,6 +88,23 @@ func main() {
 		},
 	)
 
+	if vargs.UpdateEnvironment == true {
```

Comment:
You should add the error check in front of that if block and write a separate err check within this block